update readme (#1)
DevExpress-Examples/asp-net-web-forms-grid-select-enabled-checkboxes-only: Readme.md

[![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/E3996)
[![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
<!-- default badges end -->

# Grid View for ASP.NET Web Forms - How to select all rows except disabled rows on the client side
<!-- run online -->
**[[Run Online]](https://codecentral.devexpress.com/e3996/)**
<!-- run online end -->

> Client-side [SelectAllRowsOnPage](https://docs.devexpress.com/AspNet/js-ASPxClientGridView.SelectAllRowsOnPage) and [SelectRows](https://docs.devexpress.com/AspNet/js-ASPxClientGridView.SelectRows) methods change selection state of disabled check boxes. 

Reviewer: ```suggestion
Client-side [SelectAllRowsOnPage](https://docs.devexpress.com/AspNet/js-ASPxClientGridView.SelectAllRowsOnPage) and [SelectRows](https://docs.devexpress.com/AspNet/js-ASPxClientGridView.SelectRows) methods change the selection state of disabled checkboxes. 
```

> This example illustrates how to implement **Select All** check boxes that change selection state of enabled check boxes only. To accomplish this task you should select each row in a cycle manually.

Reviewer: ```suggestion
This example illustrates how to implement **Select All** checkboxes that change the selection state of enabled checkboxes only. To do this, you should select each row in a cycle.
```
